aexpect/remote.py

```diff
@@ -403,14 +420,16 @@ def remote_login(client, host, port, username, password, prompt, linesep="\n",
 
 def wait_for_login(client, host, port, username, password, prompt,
                    linesep="\n", log_filename=None, log_function=None,
-                   timeout=240, internal_timeout=10, interface=None):
+                   timeout=240, internal_timeout=10, interface=None,
+                   preferred_authenticaton='password'):
```

Comment:
Could we also add the `user_known_hosts_file` in the end here, making it available for waiting login calls? This shouldn't harm current API and adds all options to all useful locations.

Reply:
Sure, absolutely agree.